iron-request.html

```diff
@@ -340,6 +340,7 @@
       var responseType = xhr.responseType || xhr._responseType;
       var preferResponseText = !this.xhr.responseType;
       var prefixLen = (xhr._jsonPrefix && xhr._jsonPrefix.length) || 0;
+      var warn = (console && console.warn) || function no_op() {};
```

Comment:
Ok, I wasn't sure since MDN didn't specify support for mobile browsers. Updated.